freqtrade/optimize/__init__.py

```diff
@@ -77,11 +84,17 @@ def load_tickerdata_file(
     if os.path.isfile(gzipfile):
         logger.debug('Loading ticker data from file %s', gzipfile)
         with gzip.open(gzipfile) as tickerdata:
-            pairdata = json.load(tickerdata)
+            if ujson_found is not None:
+                pairdata = ujson.load(tickerdata, precise_float=True)
```

Comment:
same here - this logic is in line 33 of this file ... can't you use the function `json_load` provided in this file ... ?

Reply:
same, actually I guess this file was lost in branches. have to push another version